### ISSUE TITLE
Uploader broken

### ISSUE THREAD

**Issue by franz-josef-kaiser** [wecodemore/wcm-avatar#55]
Currently there's a problem with the uploader where the uploader believes there already is a custom avatar present. When an existing image is deleted, the error message is broken as well and shows as in the second image.

<img width="742" alt="bildschirmfoto 2016-11-16 um 22 34 35" src="https://cloud.githubusercontent.com/assets/387666/20366784/e272aa7e-ac4c-11e6-98a9-79806a9e5cb9.png">

<img width="805" alt="bildschirmfoto 2016-11-16 um 21 43 03" src="https://cloud.githubusercontent.com/assets/387666/20366723/abbfcd22-ac4c-11e6-90f4-4438a4ecc5f8.png">

There are no errors in the console.

**Comment by franz-josef-kaiser** [comment]
No Console errors and no avatar. Still I get the second screenshot from above as result.

``` mysql
mysql> select * from wp_postmeta where meta_value = 'user_avatar';
Empty set (0.00 sec)
```

**Comment by franz-josef-kaiser** [comment]
Ok, the error is the in [`src/Services/AvatarExistsLimitService.php`](https://github.com/wecodemore/wcm-avatar/blob/90e16073fbd17a5d626b934af10fdd663ab82206/src/Services/AvatarExistsLimitService.php#L56) after the changes in #46 for #44, which was already correctly addressed in  5f22d1a by @kraftner. Seems some merge went wrong down the road.

**Comment by franz-josef-kaiser** [comment]
Fixed (again) in e2527d9
